### ISSUE TITLE
Search Radius ticketmaster.js

### ISSUE THREAD

**Issue by catchbb14** [schmitty890/eventz#10]
We may need to add a search radius to the url within the ajax request. If you do not know the exact location of an event, then no results are returned.

I am working on getting some more api keys for weatherbit api, 16 requests per day is not very much... lol

**Comment by schmitty890** [comment]
Good idea. We can create a ticket in trello and try to implement this if we have time.

Looks like weatherbit is 75 calls per hour. A couple may be needed if you're refreshing more than once a minute.
